modules/installation-configuration-parameters.adoc

```diff
@@ -1889,6 +1889,12 @@ Additional GCP configuration parameters are described in the following table:
 |The name of the GCP project where the installation program installs the cluster.
 |String.
 
+|platform:
+  gcp:
+    userProvisionedDNS:
+|Enables user-provisioned DNS instead of the default cluster-provisioned DNS solution. If you want to use this feature, you must set the `featureSet: TechPreviewNoUpgrade` parameter.
+|`Enabled` or `Disabled`. The default value is `Disabled`.
```

Comment:
This probably isn't necessary. By setting 
```
featureSet: TechPreviewNoUpgrade
```
it is enabling the feature set. It might also want to be mentioned that the user can enable this feature with 
```
featureSet: CustomNoUpgrade
featureGates:
- UserProvisionedDNS=true
```

Reply:
Reworded, thanks